common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -256,6 +267,51 @@ def has_support(self, view, functionality):
             )
         return False
 
+    def validate(self):
+        """
+        Validates the xml for correct question separation. There must be only one inputtype in every <question>.
+        This is the override of the general XBlock method.
+        """
+
+        validation = super(CapaDescriptor, self).validate()
+        validation = StudioValidation.copy(validation)
+
+        # validate xml to check questions are correctly separated
+        is_valid_xml = self.validate_xml_for_inputtypes()
+
+        if not is_valid_xml:
+            validation.set_summary(
+                StudioValidationMessage(
+                    StudioValidationMessage.WARNING,
+                    u'Incorrect problem format detected!',
```

Comment:
I like the message text that @lamagnifica suggests, but I think the action label should be an action. Perhaps something like "See documentation about formatting options". Is that too long?